src/server/ua_discovery_mdns.c

```diff
@@ -580,7 +582,7 @@ mdns_set_address_record(UA_DiscoveryManager *dm, const char *fullServiceDomain,
             int family = address->Address.lpSockaddr->sa_family;
             if(AF_INET == family) {
                 SOCKADDR_IN* ipv4 = (SOCKADDR_IN*)(address->Address.lpSockaddr); /* IPv4 */
-                mdns_set_address_record_if(dm, fullServiceDomain,
+                mdns_set_address_record_if(fullServiceDomain,
                                            localDomain, (char *)&ipv4->sin_addr, 4);
```

Comment:
Style: same like above and a few changes bellow. The parameters could be distributed differently over two lines.
```suggestion
                mdns_set_address_record_if(fullServiceDomain, localDomain,
                                           (char *)&ipv4->sin_addr, 4);
```